chart/compass/charts/director/templates/dashboard-configmap.yaml

```diff
@@ -484,28 +480,95 @@ data:
           "pointradius": 5,
           "points": false,
           "renderer": "flot",
-          "seriesOverrides": [],
+          "seriesOverrides": [
+            {
+              "alias": "Total",
+              "color": "#3274D9"
+            },
+            {
+              "alias": "Failed",
+              "color": "#F2495C",
+              "dashes": true,
+              "spaceLength": 10
+            },
+            {
+              "alias": "Completed",
+              "color": "#56A64B"
+            },
+            {
+              "alias": "Scheduled",
+              "color": "#828282"
+            },
+            {
+              "alias": "In progress",
+              "color": "#F1F14C"
+            }
+          ],
           "spaceLength": 10,
           "stack": false,
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(compass_director_customer_adoption_quantity)",
+              "expr": "sum(compass_system_fetcher_operations_total_quantity)",
               "format": "time_series",
               "instant": false,
               "interval": "10s",
               "intervalFactor": 1,
```

Comment:
Is there a reason for the interval factor of the "Total" time series to be set to 1? As far as I see for the other time series the value is 2, so that there will be a slightly delay between the synchronised values for Total and the remaining ones.